### PR TITLE
introduced rhel81 solver

### PR DESCRIPTION
**Pull request by goern**
introduced rhel81 solver, and fixed rhel80 solver (by building from earlier tag)

Signed-off-by: Christoph Görn <goern@redhat.com>

### CODE REVIEW

**Comment by harshad16** [comment]
LGTM, 
@fridex  PTAL, i think we would have to change the`python -m pip install` in this PR as well. WDYT?

**Comment by harshad16** [comment]
regate

**Comment by fridex** [comment]
> LGTM,
> @fridex PTAL, i think we would have to change the`python -m pip install` in this PR as well. WDYT?

+1 on my side

File: openshift/buildConfig-template.yaml [hunk 1]
```diff
@@ -263,13 +263,87 @@ objects:
...
+          USER 0
+          RUN useradd -m solver -s /sbin/nologin -c "Thoth Solver User"
+
+          USER ${USERID}
+          RUN mkdir -p ${THOTH_SOLVER_TMP_DIR}
+
+          # Install thoth-solver itself
+          COPY ./ ${THOTH_SOLVER_TMP_DIR}
+          RUN cd ${THOTH_SOLVER_TMP_DIR} &&\
+              pip3 install --upgrade setuptools pip &&\
```
Comment (harshad16) [inline openshift/buildConfig-template.yaml]: @goern can you change this to :
```suggestion
              pip3 install --upgrade setuptools pip &&\
    use
             `/usr/bin/python3.6 -m pip install --upgrade setuptools pip`

```

**Comment by harshad16** [comment]
regate

**Pull request merged**